logexporter/cluster/log-dump.sh

```diff
@@ -925,6 +925,11 @@ function main() {
   fi
 
   dump_logs
+  
+  if [[ "${SKIP_DUMP_NODE_INFO:-}" == "true" ]]; then
```

Comment:
Putting the return here would skip everything else that is executed after `dump_node_info` is called which I think is not what we want to achieve here.

To prevent that, I'd suggest putting this if statement in the beginning of the `dump_node_info` function.